### ISSUE TITLE
[COGNITION] 508-enhancement: CONSIDER adding aria-labelledby attributes to emergency care exclusion lists

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#15102]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I would like to hear a custom label for the exclusions to Express Care so I better understand what Express Care does **not** treat, and how I might get help if I am having an emergency or would like to speak to someone at the Veterans' Crisis Line.

## Details

The Express Care can’t provide emergency help has two lists that contain important information, but don't have any headings nearby. I'd like to add unique IDs to each `<p>` tag and an `aria-labelledby` attribute to the lists, so the lists have meaningful names.

## Acceptance Criteria

- [ ] Lists are read aloud by the `<p>` tag text by screen readers
- [ ] Zero axe-core violations detected

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Environment

* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/new-express-care-request

## Solution (if known)

```html
<!-- Would need to repeat this pattern for the one bullet point list about contacting the Veterans' Crisis Line.
<div class="usa-alert-text">
  <p id="UNIQUE_ID">Call 911 or go to the nearest emergency room now if you have any of these symptoms:<p>
  <ul aria-labelledby="UNIQUE_ID">
    ...
  </ul>
</div>
```

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2020-10-20 at 9 32 15 PM](https://user-images.githubusercontent.com/934879/96668340-94b98700-1320-11eb-8fa0-05acfa97b424.png)

**Comment by grossb1** [comment]
@1Copenut This has been deployed to staging and is ready to test.

**Comment by 1Copenut** [comment]
Retested with VO on Staging. Looks good, moving this to close.
